### ISSUE TITLE
If title auto-generation is off, the chat title is always "New chat"

### ISSUE THREAD

**Issue by disser** [open-webui/open-webui#6478]
# Bug Report

## Installation Method

docker

## Environment

- **Open WebUI Version:** v0.3.35
- **Ollama (if applicable):** N/A

- **Operating System:** container
- **Browser (if applicable):** Firefox 131

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

Chat name with title auto-generation off is the beginning of the first input to the chat.

## Actual Behavior:

Chat name with title auto-generation off is "New chat"

## Description

**Bug Summary:**
I recently changed my image in docker-compose.yml from ghcr.io/open-webui/open-webui:main to ghcr.io/open-webui/open-webui:v0.3.35 and now all my chats are "New chat" rather than the first prompt text.

I think the last time I had done `docker pull` was 2024-08-21.   I may do some bisecting to see if I can figure out where the change specifically happened.

## Reproduction Details

**Steps to Reproduce:**
Go to preferences and turn off title auto-generation, create new chat.

## Logs and Screenshots

**Browser Console Logs:**
N/A

**Docker Container Logs:**
N/A

**Screenshots/Screen Recordings (if applicable):**
N/A

## Additional Information

N/A

**Comment by delfireinoso** [comment]
I have the same result on a docker installation, on OpenWebUi 3.35 but it's from 3.32 or early. It worked before 3.30

**Comment by tjbck** [comment]
Fixed on dev!

**Comment by ohgodhowdidthisgethereiamnotgoodwithcom** [comment]
i think there might be a regression of this in v0.6.33. i see the logic for when/how titles get updated in `backend/open_webui/utils/middleware.py` was changed around a bit... i don't have time to capture debug info right now, but wanted to leave a note here.

seeing this behavior with external OpenAI chats as well as local ollama chats, i have title autogeneration disabled completely. before updating to v0.6.33 this worked as expected.

**Comment by silentoplayz** [comment]
Confirmed: I tested the behavior myself.

- With the Docker image tagged `0.6.32` and “Title Auto-Generation” turned off, the chat title is populated with the text of the user’s first message.
- After stopping the container, switching the image tag to `dev` (latest digest pulled just now), and restarting, the identical “Title Auto-Generation” setting now leaves every new conversation titled “New Chat” regardless of the first message sent.

This reproduces the regression described by @ohgodhowdidthisgethereiamnotgoodwithcom.
